jsonnet/github_action.jsonnet

```diff
@@ -50,19 +50,47 @@ local libJob(name) = {
   name: 'Generate ' + name + ' Jsonnet library and docs',
   needs: ['build', 'repos'],
   'runs-on': 'ubuntu-latest',
+
   steps: [
     { uses: 'actions/checkout@v2' },
+    {
+      uses: 'dorny/paths-filter@v2',
+      id: 'filter',
+      with: {
+        filters: |||
+          workflows:
+            - '.github/**'
+            - 'bin/**'
+            - 'Dockerfile'
+            - 'go.mod'
+            - 'go.sum'
+            - 'jsonnet/**'
+            - 'main.go'
+            - 'Makefile'
+            - 'pkg/**'
+            - 'scripts/**'
+            - 'tf/**'
+            - 'libs/%s/**'
```

Comment:
The most important check is this one. Most of the time we only update one lib in `libs/`, so it should not run that many steps.